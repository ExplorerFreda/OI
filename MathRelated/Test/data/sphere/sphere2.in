4
9622.000000 110.500000 9655.000000 16620.000000
19332.500000 12661.000000 18122.500000 1142.500000
18168.500000 4753.500000 10106.000000 15688.000000
8138.000000 16273.000000 1140.500000 17235.500000
7420.500000 15095.000000 12820.500000 12266.500000
