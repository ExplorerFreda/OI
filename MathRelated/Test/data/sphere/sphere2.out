10704.427 5675.613 -1082.948 -2252.423
